src/renderer/component/inviteNew/view.jsx

```diff
@@ -47,8 +47,11 @@ class FormInviteNew extends React.PureComponent {
             }}
           />
         </FormRow>
-        <div className="card__actions">
-          <Submit label="Invite" disabled={isPending} />
+
+        <div className="card__content">
```

Comment:
I don't think I'm a fan of this. Before it was super simple to add a list (or single) button. It was one `card__actions` class that had a `padding-top`. I think this could work, but it just takes more effort. A good example is on the wallet page. I see three buttons, with three different top/bottom spacings

<img width="600" alt="screen shot 2018-11-15 at 5 05 02 pm" src="https://user-images.githubusercontent.com/16882830/48584752-a1453180-e8f8-11e8-836e-079d3c6ef798.png">

Reply:
I just inspected the code and the buttons themselves don't have any spacing. The container they are in also don't have spacing unless they are inside another class. I could remove `.form-row--padded` from the address in **Receive Credits** (which has top/bottom spacing) to reduce the vertical spacing.

Other than that, I don't see anything wrong with taking the top-down approach to making layouts (pushing down rather than up).

Reply:
My comment was more aimed at the inconsistency of the button spacings. It seems like this still isn't fixed. The `Claim Rewards` button has a large amount of space above, and small amount of space below. The `Get New Address` button has a large amount of space on top and bottom. And the `Full History` button has a small amount of space on top and bottom.

Reply:
> The container they are in also don't have spacing unless they are inside another class.

This seems like it could cause issues. Why not just give the base level class the spacing instead of only adding it if it's inside of another class?

Reply:
These buttons are also in different contexts.

- `Claim Rewards` is below a header. Everything below a header has the same spacing.
- `Get New Address` is below a copyable input. That spacing ensures you don't click on something unintentionally.
- `Full History` is below a table and _pertains to that table_. Why would it have large spacing above it? The spacing below it is consistent with every single card.

If you want all the buttons to have "consistent spacing", they should all be in the same contexts. But they're not. Because it doesn't make sense.

Reply:
I understand not having a large space on the `Full History` button. I think that is fine.

For `Get New Address` and `Claim Rewards`, they are both directly above help text, but have two different spacings. Looks like one is `p.help` and the other is `div.help`. Might just be a typo.